src/libraries/System.Text.RegularExpressions/tests/Regex.Match.Tests.cs

```diff
@@ -1242,5 +1244,51 @@ public void Synchronized()
 
             AssertExtensions.Throws<ArgumentNullException>("inner", () => System.Text.RegularExpressions.Match.Synchronized(null));
         }
+
+        public static IEnumerable<object[]> UseRegexConcurrently_ThreadSafe_Success_MemberData()
+        {
+            foreach (TimeSpan timeout in new[] { Timeout.InfiniteTimeSpan, TimeSpan.FromMinutes(1) })
```

Comment:
curious why varying this parameter is interesting?

Reply:
It's the exact state I called out on Teams as being buggy in the DFA code? :)

Reply:
ah, I didn't see that comment.